utils/docker/examples/daos-base/el8/Dockerfile

```diff
@@ -9,16 +9,16 @@
 # - BUST_CACHE               Manage docker building cache (default undefined).  To invalidate the
 #                            cache, a random value such as the date of day shall be given.
 # - DAOS_REPOS               Space separated list of repos needed to install DAOS (default
-#                            "https://packages.daos.io/v2.2/EL8/packages/x86_64/")
+#                            "https://packages.daos.io/v2.4/EL8/packages/x86_64/")
 # - DAOS_GPG_KEYS            Space separated list of GPG keys associated with DAOS repos (default
 #                            "https://packages.daos.io/RPM-GPG-KEY")
 # - DAOS_REPOS_NOAUTH        Space separated list of repos to use without GPG authentication
 #                            (optional)
-# - DAOS_VERSION             Version of DAOS to use (default "2.2.0-4.el8")
-# - DAOS_CLIENT_UNAME        User name of the client (mandatory)
-# - DAOS_CLIENT_UID          User id of the client (mandatory)
-# - DAOS_CLIENT_GNAME        Group name of the client (mandatory)
-# - DAOS_CLIENT_GID          Group id of the client (mandatory)
+# - DAOS_VERSION             Version of DAOS to use (default "2.4.0-2.el8")
+# - DAOS_CLIENT_UNAME        User name of the client (default "root")
+# - DAOS_CLIENT_UID          User id of the client (default "0")
+# - DAOS_CLIENT_GNAME        Group name of the client (default "root")
+# - DAOS_CLIENT_GID          Group id of the client (default "0")
```

Comment:
I'm resistant to this, the client doesn't need to be root and defaulting to a privileged account when one isn't required strikes me as bad practice.  If a user wants to use root/0 then that's on them, as a default I thought we should avoid it however - even if it does raise the barrier to entry.